app/views/password_resets/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :heading, "Forgot your password?" %>
+
+<%= form_with url: password_resets_path, class: "flex flex-col space-y-4 w-80" do |f| %>
+  <%= f.label :email, class: "input input-bordered flex items-center justify-between" do %>
+    <span>Email <span class="">*</span></span>
+    <%= f.email_field :email,
+      autofocus: true,
+      placeholder: "Email",
+      class: "w-44 border-0 dark:text-black"
+    %>
+  <% end %>
+
+  <%= f.submit 'Send reset email',
```

Comment:
Let's title case these button labels since that's how it looks like I did the others (not super deliberate, just making it consistent):

```suggestion
  <%= f.submit "Send Reset Email',
```